jobs/fetch-universe-metadata/Dockerfile

```diff
@@ -1,12 +1,22 @@
 FROM mcr.microsoft.com/azure-functions/python:4-python3.12
 
-# Install relevant python libraries
+# to be added when volume is not mounted
+# COPY . /home/site/wwwroot/
+# RUN pip install -r /home/site/wwwroot/requirements.txt
 COPY requirements.txt requirements.txt
-
 RUN pip install -r requirements.txt
 
+RUN apt-get update && \
+    apt-get install -y curl apt-transport-https gnupg && \
```

Comment:
These 3 packages are already included by default in the Debian base img. Is there value in keeping these instructions here?